crates/metadata-server/src/raft/store.rs

```diff
@@ -748,7 +748,9 @@ impl Member {
             return;
         }
 
-        let nodes_config = Metadata::with_current(|m| m.nodes_config_ref());
+        let metadata_nodes_config = Metadata::with_current(|m| m.nodes_config_ref());
```

Comment:
Why not move this to the `latest_nodes_configuration` itself?

Reply:
Because methods cannot return references to values they own.

Reply:
😅